its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/SslTest.java

```diff
@@ -450,6 +459,18 @@ private AnalysisResult validateAnalysis(AnalysisContext context, HttpsReversePro
     return result;
   }
 
+  private void assertScannerEngineSuccessfulKeystore(String logs, String trustStorePath, String trustStorePassword) {
+    String trustStorePathRegex = Stream.of(trustStorePath.split("[/\\\\]"))
```

Comment:
I don't think you need to escape `\` in a character group.
```suggestion
    String trustStorePathRegex = Stream.of(trustStorePath.split("[/\\]"))
```

Reply:
`"[/\\]"` is a syntax error 😄